### PR TITLE
Added new models, services and routes

### PR DESCRIPTION
**Pull request by oribermudez**
This change is [Reviewable](https://reviewable.io/reviews/oribermudez/revitalize/18)

### CODE REVIEW

File: database/models/Therapist.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const therapistSchema = new Schema({
+  name: String,
+  email: String,
+  phone: String,
+  location: String,
```
Comment (MattYackel) [inline database/models/Therapist.js:9]: What is location vs address? Is it meant to be the location of the business? Or just leftover copypaste :)
massagesTaken as well, shouldn't need to track that for therapist. Unless it is meant to be like masagesProvided or something

**Review by MattYackel — COMMENTED** [review]
Comment in therapist model

File: database/models/Therapist.js (hunk 1 above)
Comment (oribermudez) [reply database/models/Therapist.js:9]: good catch! I copied and pasted some of the entities we have on the DB and wasn't really thorough about it, I will fix those and let you know! 
I'm glad you are actually reviewing code, this makes me so happy! thanks Matt

**Review by MattYackel — COMMENTED** [review]
> Reviewed 28 of 28 files at r1, 6 of 6 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/oribermudez/revitalize/18)* status: all files reviewed, 1 unresolved discussion (waiting on @oribermudez)

**Review by MattYackel — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/oribermudez/revitalize/18)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @oribermudez)

**Pull request merged**